server/src/main/java/com/google/coffeehouse/common/StorageHandler.java

```diff
@@ -14,40 +14,109 @@
 
 package com.google.coffeehouse.common;
 
+import static com.google.cloud.spanner.TransactionRunner.TransactionCallable;
+
 import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.Mutation;
 import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
 import com.google.cloud.spanner.Statement;
+import com.google.cloud.spanner.TransactionContext;
+import com.google.cloud.spanner.Value;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 /**
 * The StorageHandler class holds the query functions to get information from the database.
 */
 public class StorageHandler {
   public static final String NO_PRONOUNS = "No pronouns";
   public static final String PERSON_DOES_NOT_EXIST = "This person does not exist in the database.";
-  public static final String ERROR_MORE_THAN_ONE_PERSON = "ERROR: More than one person per user ID.";
+  public static final String ERROR_MORE_THAN_ONE_PERSON = "More than one person per user ID.";
 
   public static final String CLUB_DOES_NOT_EXIST = "This club does not exist in the database.";
-  public static final String ERROR_MORE_THAN_ONE_CLUB = "ERROR: More than one club per club ID.";
+  public static final String ERROR_MORE_THAN_ONE_CLUB = "More than one club per club ID.";
 
   public static final String NO_AUTHOR = "No author";
   public static final String NO_ISBN = "No ISBN";
   public static final String BOOK_DOES_NOT_EXIST = "This book does not exist in the database.";
-  public static final String ERROR_MORE_THAN_ONE_BOOK = "ERROR: More than one book per book ID.";
+  public static final String ERROR_MORE_THAN_ONE_BOOK = "More than one book per book ID.";
+  
+  public static Person createPersonFromId(String userId) {
+    Spanner spanner = StorageHandlerSetup.createSpannerService();
+    DatabaseClient dbClient = StorageHandlerSetup.createDbClient(spanner);
+    Person person = StorageHandler.getPersonQuery(dbClient, userId);
+    return person;
+  }
+
+  public static Club createClubFromId(String clubId) {
+    Spanner spanner = StorageHandlerSetup.createSpannerService();
+    DatabaseClient dbClient = StorageHandlerSetup.createDbClient(spanner);
+    Club club = StorageHandler.getClubQuery(dbClient, clubId);
+    return club;
+  }
+  
+  public static void addPersonToClub(String userId, String clubId) {
+    Spanner spanner = StorageHandlerSetup.createSpannerService();
+    DatabaseClient dbClient = StorageHandlerSetup.createDbClient(spanner);
+    List<Mutation> mutations = new ArrayList<>();
+    try {
+      mutations.add(
+      Mutation.newInsertBuilder("Memberships")
+        .set("userId")
+        .to(userId)
+        .set("clubId")
+        .to(clubId)
+        .set("membershipType")
+        .to(Membership.MEMBER)
+        .set("timestamp")
+        .to(Value.COMMIT_TIMESTAMP)
+        .build());
+    dbClient.write(mutations);
+    } catch (Exception e) {
```

Comment:
This should check whether the failure is because the person already exists. There are other potentially reasons for this to fail, and we shouldn't throw an inaccurate exception.